### ISSUE TITLE
Use Workbox for service workers

### ISSUE THREAD

**Issue by jtiala** [jtiala/reducks-starter#17]
Currently used libraries are deprecated.

![Screenshot 2019-09-05 at 13 29 57](https://user-images.githubusercontent.com/1576782/64334514-59a3a980-cfe1-11e9-939f-a3fa11350f37.png)

**Comment by Dan2D** [comment]
Hey! 
Would, I be able to take on this issue?

**Comment by jtiala** [comment]
Sure thing! Thanks for the help and happy hacktoberfest!

**Comment by Dan2D** [comment]
Submitted a PR.

Couple of things
Many of the options used in the sw-precache are no longer valid or are automatically taken care of by workbox and so much of the options were removed in webpack.config.

Also changed navigate to fallback path to just point back to index.html, was getting errors with public path.

Tested in chrome and it registers service worker and precaches assets,

Also might want to think about splitting webpack config into common, dev, and prod, I'm running a windows machine and NODE_ENV=..... command doesn't work in package.json scripts so had to change it when I ran, but changed it back before commit

Let me know if anything needs changing!
-Dan
